lightning/src/util/events.rs

```diff
@@ -201,6 +206,30 @@ impl_writeable_tlv_based_enum_upgradable!(ClosureReason,
 	(12, OutdatedChannelManager) => {},
 );
 
+/// Struct to `Display` `CounterpartyForceClosed::peer_msg` using `PrintableString`
+/// Print the peer_msg using Display to sanitize it (remove control characters)
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
Why add a newtype if this is only suppose to be for `Display`?

Reply:
Ah, right, we can't give visibility to enum variant fields. Wonder if we should just make a `SanitizedString` type, or the like, so we don't need to add a newtype each time we need this behavior. Would also be less verbose.

The other uses of `PrintableString` are as part of an interface between the underlying representation, which isn't necessarily a string, thus why it wraps a reference.

Reply:
This suggestion makes a lot of sense, I moved the struct to `crate::util::string` and called it SanitizedString